ipm_library/ipm_library/ipm.py

```diff
@@ -148,6 +148,8 @@ def map_points(
         if not np.any(plane_msg.coef[:3]):
             raise InvalidPlaneException
 
+        assert np.shape(points)[1] == 2, 'Points must be in the form of a nx2 numpy array'
```

Comment:
Why not `points.shape[1]`?

Reply:
Due to my lack of experience with numpy :laughing: